templates/data/news-211102a.md

```diff
@@ -0,0 +1,9 @@
+# news-tags
```

Comment:
Thanks for including these. Please move to the bottom of the page and wrap in HTML comment tags `<!-- a normal html comment -->`

Reply:
Use the title "News" on line 1 as done on featured data pages.